lib/driver.rb

```diff
@@ -0,0 +1,74 @@
+require 'csv'
+require_relative 'csv_record'
+
+VALID_STATUS = [:AVAILABLE, :UNAVAILABLE]
+
+module RideShare
+  class Driver < CsvRecord
+
+    attr_reader :name, :vin, :status, :trips
+
+    def initialize(
+        id:,
+        name:,
+        vin:,
+        status: :AVAILABLE,
+        trips: []
+    )
+
+      super(id)
+      @name = name
+      @vin = vin
+      @status = status
+      @trips = trips
+
+      unless @vin.length == 17
+        raise ArgumentError, "Invalid vehicle identification number"
```

Comment:
It's helpful to include the invalid value in your messages:

```suggestion
        raise ArgumentError, "Invalid vehicle identification number: #{@vin}"
```